website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -6,38 +6,44 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
 
 :::note Announcement
 
-The [user API tokens](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). We recommend rotating your existing user tokens with PATs. There are no deprecation plans for user API tokens at this time; we will give ample notice when that timeline has been determined. 
+[User API keys](#user-tokens) are being replaced by [account-scoped Personal Access Tokens(PATs)](#account-scoped-personal-access-tokens). 
+
+We recommend rotating your existing user API keys with PATs. There are no deprecation plans for user API keys at this time; we will give ample notice when that timeline has been determined. 
 
 Cloud CLI config files do not need to be updated at this time. You will be notified when you need to re-download your configs.
 
-The current API key is located under **Personal Settings → API Key.** 
+The current user API key is located under **Personal Settings → API Key.** 
 
 Please [contact support](/docs/dbt-support#dbt-cloud-support) with any questions or concerns.
 
 :::
 
-## Account-scoped personal access tokens
+## Account-scoped Personal Access Tokens
 
 :::info New 
 
-On Feb 7, 2024, we introduced a new type of token for individual users called personal access tokens. Note that these differ from [Service Tokens or API Keys](/docs/dbt-cloud-apis/authentication#types-of-api-access-tokens). Before this release, user API keys were the only way to access dbt Cloud API on behalf of the user. These API Keys were user-specific and were not scoped to an account. To enhance the security of dbt Cloud, we are moving away from this model to account-specific tokens. 
+On Feb 7, 2024, we introduced a new type of token for individual users called Personal Access Tokens. Note that these differ from [Service Tokens or user API Keys](/docs/dbt-cloud-apis/authentication#types-of-api-access-tokens). 
+
+Before this release, User API keys were the only way to access dbt Cloud API on behalf of the user. 
```

Comment:
This sentence feels very linked to the previous paragraph. Is there a reason you separated it from the paragraph?
```suggestion
Before this release, user API keys were the only way to access dbt Cloud API on behalf of the user. 
```

Reply:
Added a break for readability, the large block felt difficult to parse:
<img width="826" alt="Screenshot 2024-05-29 at 9 57 21 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/18703280/e56ace77-420c-4895-9f0b-667f3cd5e2ed">

Thoughts on keeping the first paragraph break below if i remove that 2nd paragraph break? Or would it be better to keep it as it was before?
<img width="826" alt="Screenshot 2024-05-29 at 10 02 21 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/18703280/4a1f1b84-4cb7-408c-aa5c-9e68a47b390f">